app/views/completed_applications/index.html.erb

```diff
@@ -4,11 +4,15 @@
 
 <%= render partial: 'shared/subnavigation' %>
 
-<% if @applications.nil? %>
-  <%= render partial: 'datastore_unavailable' %>
-<% else %>
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <% if @applications.nil? %>
+      <%= render partial: 'datastore_unavailable' %>
+    <% elsif @applications.empty? %>
+      <h2 class="govuk-heading-m">
+        <%= t('.no_records', status: (params[:q] ||= 'submitted')) %>
```

Comment:
I feel this may be dangerous. We are blindly propagating whatever value is in the `q` query param to the locales. For once it will blow up for anything like `foobar`, but there might be other side effects.

I think the way to handle this is to have a helper method `status_filter` in the dashboard controller (as that is the base controller the other inherit from) that is implemented in both controllers, in one it just return `in_progress` for instance, and in the completed controller it takes and sanitises the `param[:q]` (then we don't need that code in the GetApplications service anymore, we just pass the proper sanitised value to the service object).

Then in locales we have add to the shared errors a new section `no_records` for instance and declare the 3 different messages. Propagating the raw `q` value doesn't work if we translate to welsh.